src/components/WriteButton/WriteButton.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+const StyledWriteButton = styled.button`
+  align-items: center;
+  background-color: var(--white);
+  border: none;
+  border-radius: 40px;
+  box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15),
+    0px 1px 2px rgba(0, 0, 0, 0.15);
+  color: var(--mako);
+  cursor: pointer;
+  display: flex;
+  font-weight: 500;
+  font-size: 1rem;
+  height: 3rem;
+  justify-content: center;
+  margin: 0 1rem;
+  padding: 0.75rem, 1.25rem, 1.25rempx, 1.25rem;
```

Comment:
Check if that use rem or px

Reply:
What you think about split css code of component declaration file ? something like styled.js

Reply:
I was thinking about that, actually. I suggest creating a folder styled in which we will have a folder for each kind of component so taking into account the current component we would result in something like `src/styled/Buttons/WriteButton/`

Reply:
Tell me what you think about it or if you would prefer this [approach](https://www.es6.today/2019/04.06/organizing-styled-components/)

Reply:
I prefer the component file beside the styling file, @marclerodrigues can speak better about that

Reply:
When dealing with complex systems I prerer to have a separate folder for all styling, but in small apps the preferable approach is to put the style right next to the component declaration.